apps/api/pages/api/bookings/_post.ts

```diff
@@ -206,7 +207,7 @@ async function handler(req: NextApiRequest) {
   const { userId, isAdmin } = req;
   if (isAdmin) req.userId = req.body.userId || userId;
 
-  return await handleNewBooking(req);
+  return await handleNewBooking(req, getBookingDataSchemaForApi);
```

Comment:
Since the API handler is the odd one out. It should only pass the desired schema instead of doing `isNotAnApiCall: true` in all other calls.